src/lib/planningApplication/progress.tsx

```diff
@@ -40,6 +40,12 @@ import {
   slugify,
 } from "@/util";
 import { contentImportantDates } from "./date";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import isSameOrAfter from "dayjs/plugin/isSameOrAfter";
+dayjs.extend(utc);
+dayjs.extend(isSameOrAfter);
```

Comment:
We're not using `isSameOrAfter` in this file so we don't need this plugin enabled ([docs](https://day.js.org/docs/en/plugin/is-same-or-after))